.dev/compose.yml

```diff
@@ -35,7 +35,7 @@ services:
     environment:
       - CUSTOM_CONFIG_LOCATION=/config.json
       - SYNC_ENABLED=${SYNC_ENABLED:-false}
-      - DATABASE_NAME=${DATABASE_NAME_PREFIX}-2
+      - DATABASE_NAME=${DATABASE_NAME_PREFIX:-connector}-2
```

Comment:
looks like this was committed by accident

Reply:
Na I just copied it from connector1 without this, connector 2 won't start if no DATABASE_NAME_PREFIX is provided

Reply:
IC!